NETWORK_PROTOCOL.md

```diff
@@ -1 +1,102 @@
-Placeholder for network protocol.
+Status: this is a rough draft, intended to get some ideas out there. 
+
+Missing: how to run on top of HTTP/2. Possibly the structure or semantics of publisher name strings.
+Might be removed: extension support.
+Need to choose serialization method (protobuf, msgpack, other?)
```

Comment:
Are you suggesting we define one, or just that the bytes should be serializable using any of these mechanisms?